make/autoconf/lib-freetype.m4

```diff
@@ -192,6 +196,21 @@ AC_DEFUN_ONCE([LIB_SETUP_FREETYPE],
               [$FREETYPE_BASE_DIR/lib], [well-known location])
         fi
 
+        if test "x$FOUND_FREETYPE" != "xyes" ; then
+          LIB_CHECK_POTENTIAL_FREETYPE([$FREETYPE_BASE_DIR/include],
+              [$FREETYPE_BASE_DIR/lib/$OPENJDK_TARGET_CPU-linux-gnueabihf], [well-known location])
```

Comment:
Do we really need to explicitly list all the gnu* variants here? Isn't the subdir simply the value of openjdk-target? At least it looks that way in the submit.yml.

Reply:
I think that value should be accessible from OPENJDK_TARGET_AUTOCONF_NAME.

Reply:
Unfortunately, AUTOCONF_NAME is a quadruplet. For example, for ARM `OPENJDK_TARGET_AUTOCONF_NAME=arm-unknown-linux-gnueabihf`, but the subdirectory is `arm-linux-gnueabihf`. I don't see how can I get the original `--openjdk-target` provided by user here. Seems to me it is not passed to autoconf?

Reply:
What I can do is parse the ABI from the original string, and then reconstruct the triplet as `$OPENJDK_TARGET_CPU-linux-$OPENJDK_TARGET_ABI`. See the update.

Reply:
This is certainly much better. Would it make sense to replace "linux" with $OPENJDK_TARGET_OS as well, just to avoid having any hard coded values there?

Reply:
Yeah, I can, see update.